Allegro fix for new version of glibc

fabriziodiblasi opened the pull request:
Gentili sviluppatori,

Ho riscontrato un bug dovuto alla nuova versione di glibc installata su ubuntu 18.10.
È risolvibile inserendo la seguente linea di codice:
#define ALLEGRO_NO_FIX_ALIASES /* fix for the new version of glibc */

saluti




[examples.zip](https://github.com/glipari/ptask/files/2698086/examples.zip)

PierreCaruyer committed:
committing changes on dle_timer

PierreCaruyer committed:
committing changes on dle_timer

Commit:
Merge pull request #5 from PierreCaruyer/master

deadline exception mechanism added

Commit:
adding gitignore

Commit:
Adding install commands

Commit:
Merge branch 'master' of github.com:glipari/ptask

mtango committed:
Formatted everything using clang-format

 clang-format -style="{BasedOnStyle: llvm, IndentWidth: 4}"  -i *.c
 clang-format -style="{BasedOnStyle: llvm, IndentWidth: 4}"  -i *.h

mtango committed:
Fix gitignore

mtango committed:
tests/time.c: Fix bad assert

mtango committed:
Fix call to dle_timer_initialized

mtango committed:
Fix spelling: esclusion -> exclusion

mtango committed:
activation_example/testTaskFun.c: fix arguments to init()

mtango committed:
examples: Fix uses of the deprecated ptask_create

replace them with ptask_create_prio

mtango committed:
Fix int check.

mtango committed:
Fix use before initialization.

mtango committed:
Fix bad initialization

Commit:
Merge branch 'mtango-master'

Commit:
test exception removed for the moment; the dle library does not work properly

Commit:
Fix for allegro and new glibc

glipari commented:
This is not the way a pull request should be done. Please look at this: 

https://help.github.com/articles/creating-a-pull-request-from-a-fork/ 

You must "fork" my repo, then make the changes on your forked repo, commit and push. Finally, you open a pull request, which asks for a merge of you modifications to my repo.

Commit:
adding ptaskTracer

Commit:
updating readme

Commit:
Merge pull request #10 from ael-mess/master

Integrating ptaskTracer

glipari committed:
Adding Amine in the list of authors

glipari committed:
Adding a link to ptaskTracer

glipari committed:
change version number

glipari committed:
adding an example for the tracer, using fixed priority on a single processor

glipari committed:
minor fixes

glipari committed:
Adding data.txt to the repo

FloraDziedzic committed:
Ajout d'un nouveau champ dans les tracepoints : l'offset pour connaître calculer la date de réveil de la tâche
 Cet offset sera stocké dans la structure des tâches périodiques

Nouveau tracepoint lors de l'activation d'une tâche apériodique lors de l'appel à ptask_activate() pour les dates de réveil des tâches apériodiques

FloraDziedzic committed:
ajout de la date d'activation "réelle" actuelle pour tracer les dates de réveil des tâches. Pour ne pas perturber le fonctionnement du paramètre ptask.at, ce nouvel élément est ajouté dans les paramètres de la tâche

Commit:
Merge pull request #14 from FloraDziedzic/master

Modification des tracepoints pour PtaskTracer pour les dates de réveil

glipari committed:
Calibration via environment variable

glipari committed:
Forcing 32 bit computation on calibration.
